.travis.yml

```diff
@@ -6,6 +6,7 @@ python:
   - "3.6"
 install:
   - "pip3 install -r requirements.txt"
+  - "pip install coveralls"
```

Comment:
All other requirements are installed for python3 (line 8 uses pip3), why are you not installing coveralls for python3 ?

Reply:
Hey, I will just update the line with `pip3` to make everything uniform. However the command is tantamount to `pip3` in a python 3 environment.